cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -473,7 +473,7 @@ func getStringEventsForPod(cs corev1client.EventsGetter, pod *corev1.Pod) string
 	var res string
 	events, err := cs.Events(pod.Namespace).List(context.TODO(), metav1.ListOptions{FieldSelector: fmt.Sprintf("involvedObject.name=%s", pod.Name), TypeMeta: metav1.TypeMeta{Kind: "Pod"}})
 	if err != nil {
-		return err.Error()
+		return fmt.Sprintf("can't retrieve events for pod %s/%s: %s", pod.Namespace, pod.Name, err.Error())
```

Comment:
Can we please make this even more robust by checking if the pod pointer is nil before calling the "cs.Events"- then catch it and return err also?